src/state/reducer.ts

```diff
@@ -41,18 +59,78 @@ const preferencesPersistConfig = {
   blacklist: ['featureFlags'],
 }
 
+const txHistoryPersistConfig = {
+  key: 'txHistory',
+  storage: localforage,
+  version: Math.max(...Object.keys(clearTxHistoryMigrations).map(Number)),
+  migrate: createMigrate(clearTxHistoryMigrations, { debug: false }),
+}
+
+const portfolioPersistConfig = {
+  key: 'portfolio',
+  storage: localforage,
+  version: Math.max(...Object.keys(clearPortfolioMigrations).map(Number)),
+  migrate: createMigrate(clearPortfolioMigrations, { debug: false }),
+}
+
+const opportunitiesPersistConfig = {
+  key: 'opportunities',
+  storage: localforage,
+  version: Math.max(...Object.keys(clearOpportunitiesMigrations).map(Number)),
+  migrate: createMigrate(clearOpportunitiesMigrations, { debug: false }),
+}
+
+const nftPersistConfig = {
+  key: 'nft',
+  storage: localforage,
+  version: Math.max(...Object.keys(clearNftsMigrations).map(Number)),
+  migrate: createMigrate(clearNftsMigrations, { debug: false }),
+}
+
+const snapshotPersistConfig = {
+  key: 'snapshot',
+  storage: localforage,
+  version: Math.max(...Object.keys(clearSnapshotMigrations).map(Number)),
+  migrate: createMigrate(clearSnapshotMigrations, { debug: false }),
+}
+
+const localWalletSlicePersistConfig = {
+  key: 'localWalletSlice',
+  storage: localforage,
+}
```

Comment:
[question]: Should we comment why there is no migration for this slice?

Reply:
c31098dba